### PR TITLE
PermaLink: Remove Permanent Link Service

### PR DESCRIPTION
**Pull request by kergomard**
At the Technical Board we had a look at the PermanentLink-Service as it seemed to us, that it was not needed anymore. I took a closer look and it actually really seems not to have a purpose anymore. It was used in a few places to show the permanent link on the Info-Page, but this is unnecessary, as we always show the permanent link in the footer.

Thus this PR proposes to remove it.

### CODE REVIEW

File: Modules/File/classes/class.ilObjFileGUI.php [hunk 1]
```diff
@@ -375,7 +375,8 @@ protected function uploadFiles(): void
         if (self::UPLOAD_ORIGIN_DROPZONE === $origin) {
             $dropzone = new ilObjFileUploadDropzone($this->parent_id);
             $dropzone = $dropzone->getDropzone()->withRequest($this->request);
-            $files = $dropzone->getData()[self::PARAM_FILES] ?? null;;
+            $files = $dropzone->getData()[self::PARAM_FILES] ?? null;
+            ;
```
Comment (nicoroeser) [inline Modules/File/classes/class.ilObjFileGUI.php]: :wink: Nitpick: extra line, caused by prior typo? Don’t think it marks an important spot in the code; let’s drop the extra semicolon (the new line 379), shall we?

Comment (kergomard) [reply Modules/File/classes/class.ilObjFileGUI.php]: Is gone now!

**Review by alex40724 — APPROVED** [review]
Great! Thanks.

**Comment by kergomard** [comment]
@smeyer-ilias , @pascalseeland , @chfsx , and @Amstutz could you please have look at this? I would like to merge it by the end of this week.
Thank you very much and best,
@kergomard

**Review by Amstutz — APPROVED** [review]
thx a lot!

**Review by chfsx — APPROVED** [review]
Thanks a lot, no objections from my side and thanks for the boyscouting (`;;`)

**Comment by kergomard** [comment]
Ok, as announced, I do merge this now.

**Pull request merged**